core/src/operations/decorators/debug.rs

```diff
@@ -13,13 +13,31 @@ pub enum DebugOptions {
     StackAll,
     /// Prints out the top n items of the stack for the current context.
     StackTop(u16),
+    /// Prints out the entire contents of RAM.
+    MemAll,
+    /// Prints out the contents of memory stored in the provided interval. Interval boundaries are
+    /// both inclusive.
+    ///
+    /// First parameter specifies the interval starting address, second -- the ending address.
+    MemInterval(u32, u32),
+    /// Prints out locals stored in the provided interval of the currently executing procedure.
+    /// Interval boundaries are both inclusive.
+    ///
+    /// First parameter specifies the starting address, second -- the ending address, and the third
+    /// specifies the overall number of locals.
+    LocalInterval(u32, u32, u32),
```

Comment:
Number of locals should not be greater than `u16` - so, we should probably use `u16`'s here instead of `u32`s.